upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -1098,6 +1098,17 @@ if [[ ${state_recorded} == "0" ]]; then
     rpm --force -Uvh "$(find "${CSM_ARTI_DIR}"/rpm/cray/csm/ -name \*goss-servers\*.rpm | sort -V | tail -1)"
     systemctl restart goss-servers
 
+    # CASMPET-6635
+    # Get the RPM versions from ncn-m001
```

Comment:
I would say "from the primary node" instead of m001.   This will be m002 on vshasta.